.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+# General summary
+
+<!-- Please describe the PR content -->
+
+# Check list
+
```

Comment:
I hesitated to put to much React specific stuff here because it would bloat the check list for the UI/UX team.
Maybe we could like to a more detailled list in an other markdown?